daml-lf/engine/src/test/scala/com/digitalasset/daml/lf/engine/EngineTest.scala

```diff
@@ -1603,13 +1603,13 @@ class EngineTest(majorLanguageVersion: LanguageMajorVersion)
         packageName = getPackageName(basicTestsPkg),
         templateId = usedDisclosedContract.templateId,
         arg = usedDisclosedContract.argument.toNormalizedValue(transactionVersion),
+        agreementText = "", // to be removed
```

Comment:
```suggestion
        // TODO: https://github.com/DACH-NY/canton/issues/15209
        //   to be removed
        agreementText = "",
```

Reply:
please follow this convention in anything in daml-lf/compiler

Reply:
Second though, since we will remove the field very soon, it is fine to go without comment at all. (I fine you let you // to be remove) here.